frontend/src/App.js

```diff
@@ -99,10 +100,31 @@ const App = () => {
       getPosts()
     })
   }
+  const [upvote_clicks, set_upvote_clicks] = React.useState(0)
```

Comment:
Snake case is fine, but using camel case (ex. `upvoteClicks` instead of `upvote_clicks`) is more of a JavaScript standard.